packages/components/src/components/SquiggleViewer/ValueWithContextViewer.tsx

```diff
@@ -233,14 +245,22 @@ export const ValueWithContextViewer: FC<Props> = ({
     };
 
     return (
-      <div className={clsx("leading-3", showColon || "mr-3")}>
-        <span
+      <div
+        className={clsx(
+          "leading-3 flex flex-row items-center",
+          showColon || "mr-3"
+        )}
+      >
+        {isRootImport && (
+          <DocumentTextIcon size={12} className="mr-1 text-violet-900" />
+        )}
+        <div
           className={clsx(!taggedName && "font-mono", headerClasses())}
           onClick={focus}
         >
-          {taggedName || name}
-        </span>
-        {showColon && <span className="text-gray-400 font-mono">:</span>}
+          {(isRootImport && exportData?.sourceId) || taggedName || name}
```

Comment:
When something is imported and then copied to a local var (not sure why someone would do that, but...), it's displayed as its source id:
<img width="1215" alt="Screenshot 2024-02-09 at 16 40 04" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/7f4f109b-a0b8-4f0d-9e15-f1c818d653e1">

I can also imagine that seeing "as" name in "Imports" tab can be useful in other cases (e.g. I'm reading the code and want to refer to things that are available).

Maybe we could show both here, "foo from hub:source1"?

Reply:
I think we should figure out some better solution here, likely a bit later on.